### ISSUE TITLE
I wish an example for the outputStruct node

### ISSUE THREAD

**Issue by Spiri0** [mrdoob/three.js#28180]
### Description

I looked in the examples and asked in the forum but so far only Berthur and I have had anything to do with the outputStruct.
Therefore I would like a simple example that demonstrates the outputStruct with two shaders whose shaders can be stored in textures with a renderTarget.

Something like that:
```
renderTarget = new THREE.RenderTarget(window.innerWidth * dpr, window.innerHeight * dpr, {count: 2});

const fragmentShader1 = wgslFn(`
  fn main1(
  ) -> vec4<f32> {
    return vec4<f32>(1, 0, 0, 1);
  }
`);

const fragmentShader2 = wgslFn(`
  fn main2(
  ) -> vec4<f32> {
    return vec4<f32>(0, 1, 0, 1);
  }
`);


const material = new MeshBasicNodeMaterial();
material.outputNode = outputStruct(
  fragmentShader1(),
  fragmentShader2(),
);

quad = new QuadMesh(material);



//in the render loop
quad.render(renderer);

renderer.setRenderTarget( renderTarget );
renderer.render( scene, camera );
renderer.setRenderTarget( null );
```

https://discourse.threejs.org/t/use-of-multiple-fragmentshaders-with-outputstruct-and-rendertarget/64287

I looked at the RenderTarget code in threejs. A lot seems to have happened over the last few releases. The RenderTarget class also seems to be designed to process multiple textures because I see an array for textures in it. But I admit that I don't yet understand the renderTarget class well enough to be able to derive an example from it.

### Solution

The solution would be a simple example code as described in the description

### Alternatives

The alternative would be a documentation, but in the end this would contain the same thing as a simple example and an example is the nicer solution

**Comment by Mugen87** [comment]
>  The RenderTarget class also seems to be designed to process multiple textures because I see an array for textures in it.

This is for MRT. Could this example help: [webgpu_multiple_rendertargets](https://threejs.org/examples/webgpu_multiple_rendertargets)

**Comment by Mugen87** [comment]
In this case, this issue can then be merged into #22403.
